src/client.js

```diff
@@ -645,7 +646,12 @@ class Client {
         var instanceKey = connectionParameters._endpoint || "";
         if (instanceKey.startsWith("http://")) instanceKey = instanceKey.substr(7);
         if (instanceKey.startsWith("https://")) instanceKey = instanceKey.substr(8);
-        const rootKey = `acc.js.sdk.${sdk.getSDKVersion().version}.${instanceKey}.cache`;
+
+        // Determine the cache root key. By default, it's ade of the 
+        const rootKeyType = connectionParameters._options.cacheRootKey;
+        let rootKey = "";
+        if (!rootKeyType || rootKeyType === "default")
+            rootKey = `acc.js.sdk.${sdk.getSDKVersion().version}.${instanceKey}.cache.`;
 
```

Comment:
I think I'm a little confused with "rootKeyType" and "rootKey".
If I understand well, if we have rootKeyType as "default", the value is in "rootKey". but if we have customised "rootKeyType", what's the value of rootKey?
maybe something like
```
 if (!rootKeyType || rootKeyType === "default")
   rootKey = `acc.js.sdk.${sdk.getSDKVersion().version}.${instanceKey}.cache.`;
else
  rootKey = rootKeyType
```
?

Reply:
There are 3 possible values:
* no value or "default" is the current behavior, i.e. all cache keys are prefixed with "acc.js.sdk.${sdk.getSDKVersion().version}.${instanceKey}.cache."
* "none" (or actually anything else for now), where cache keys are not prefixed

I'll update the comments to make it more clear